.github/workflows/package-publish.yml

```
@@ -1,25 +1,88 @@
-name: publish-npm-package
+name:  publish-npm-packages
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)